### PR TITLE
Add check for Feb 28 and 29 of the same year

### PR DESCRIPTION
**Pull request by tan75**
Add extra check to test if dates are either Feb 28 or Feb 29 of the same year
Add extra tests for intervalToDuration() to test the same

### CODE REVIEW

**Review by kossnocorp — APPROVED** [review]
Looks good!

File: src/differenceInMonths/test.js [hunk 1]
```diff
@@ -5,6 +5,41 @@ import assert from 'power-assert'
...
+    var result = differenceInMonths(start, end)
+    assert(result === 0)
+  })
+
+  it('returns the number of full months between the given dates - end of Feb', function () {
+    const start = new Date('Feb 29 2012 09:00:00')
+    const end = new Date('Feb 29 2012 10:00:00')
+    var result = differenceInMonths(start, end)
+    assert(result === 0)
+  })
```
Comment (kossnocorp) [inline src/differenceInMonths/test.js:34]: Please do the following:

1. Join these tests into a single one, as all of them have the same description
2. Move edge case tests to the end of the file, you might even wrap them into `describe('edge cases', ...` or `describe('regression tests', ...`

File: src/intervalToDuration/test.js [hunk 2]
```diff
@@ -4,8 +4,50 @@
...
+    const result = intervalToDuration({ start, end })
+    assert.deepEqual(result, {
+      years: 0,
+      months: 0,
+      days: 0,
+      hours: 1,
+      minutes: 0,
+      seconds: 0,
+    })
+  })
```
Comment (kossnocorp) [inline src/intervalToDuration/test.js:48]: Same here, join tests and move to the bottom.

Could you also fix the `Returns` -> `returns`.

**Review by kossnocorp — CHANGES_REQUESTED** [review]

File: src/differenceInMonths/test.js (hunk 1 above)
Comment (tan75) [reply src/differenceInMonths/test.js:34]: Done

File: src/intervalToDuration/test.js (hunk 2 above)
Comment (tan75) [reply src/intervalToDuration/test.js:48]: Done

**Review by kossnocorp — APPROVED** [review]
👍
